From e01816e8a20a82db8a6f39637049006a874a9176 Mon Sep 17 00:00:00 2001
From: Steve Loughran <stevel@apache.org>
Date: Tue, 23 Jan 2018 09:09:11 -0800
Subject: [PATCH 2735/2848] HADOOP-15185. Update adls connector to use the
 current version of ADLS SDK. Contributed by Atul
 Sikaria.

(cherry picked from commit f63d13f10db708228482b53883863c873c89f69c)

Change-Id: I486b664f17db82d70bde1ed91112656c572a08b3
---
 hadoop-tools/hadoop-azure-datalake/pom.xml |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/hadoop-tools/hadoop-azure-datalake/pom.xml b/hadoop-tools/hadoop-azure-datalake/pom.xml
index c4b9ef9..0797014 100644
--- a/hadoop-tools/hadoop-azure-datalake/pom.xml
+++ b/hadoop-tools/hadoop-azure-datalake/pom.xml
@@ -103,7 +103,6 @@
         </plugin>
       </plugins>
     </pluginManagement>
-
   </build>
 
   <dependencies>
@@ -111,7 +110,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-data-lake-store-sdk</artifactId>
-      <version>2.2.3</version>
+      <version>2.2.5</version>
     </dependency>
     <!--  ENDS HERE-->
     <dependency>
-- 
1.7.9.5

